src/bitmessagemain.py

```diff
@@ -342,8 +368,8 @@ def start(self):
             # Populate with hardcoded value (same as connectToStream above)
             state.streamsInWhichIAmParticipating.append(1)
 
-        if daemon == False and state.enableGUI: # FIXME redundant?
-            if state.curses == False:
+        if daemon is False and state.enableGUI:  # FIXME redundant?
```

Comment:
`if not daemon and state.enableGUI:  # FIXME redundant?` 
       `if not state.curses: `

Reply:
I'd better drop this hunk. This PR is not about style.